Framework/DataObjects/src/ScanningWorkspaceBuilder.cpp

```diff
@@ -44,13 +34,13 @@ void ScanningWorkspaceBuilder::setInstrument(
  * @param histogram A histogram with bin edges defined
  */
 void ScanningWorkspaceBuilder::setHistogram(
-    const HistogramData::Histogram &histogram) {
+    const HistogramData::Histogram histogram) {
   if ((histogram.counts().size() != m_nBins) ||
       (histogram.binEdges().size() != m_nBins + 1))
```

Comment:
Use `Histogram::size()`. `Histogram` guarantees that the bin edges are always one more than the number of bins, no need to checking.